src/Propulsion.Cosmos/ChangeFeedProcessor.fs

```diff
@@ -73,7 +73,7 @@ type ContainerId = { database: string; container: string }
 [<RequireQualifiedAccess; NoComparison; NoEquality>]
 type Discovery =
     | UriKeyAndPolicy of databaseUri:Uri * key:string * policy: ConnectionPolicy
-    | Custom of DocumentClient
+    | Custom of DocumentClient * databaseUri:Uri * policy: ConnectionPolicy
```

Comment:
Is this a bit redundant?
// databaseUri = client.ServiceEndpoint
// policy = client.ConnectionPolicy

Reply:
no, that was me just not trying ;) thanks for doing my thinking!
I'm also thinking that Equinox's cosmos.fs might benefit from a tweak or too also
will adjust

Reply:
Looks better now I've collapsed all the things, thanks for the nudge!